benchmark/misc/hidestackframes.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const common = require('../common.js');
+
+/* there should be no significant performance differences
```

Comment:
nit: `//` syntax is usually used for such comments. Also, the comment seems to be pretty obvious and could be omitted.